CONTRIBUTING.md

```diff
@@ -90,32 +88,21 @@ where you feel less comfortable or need to learn something new.
 
 **tl;dr:**
 1. Work on a feature branch
-1. [Run linting and tests often](README.md#linting--running-tests)
-1. Ensure your change has test coverage (_ideally complete coverage_)
-1. Submit a descriptive pull request
+2. Submit a descriptive pull request
 
 Once you have selected an issue and assigned it to yourself, you will start
 writing the code on an appropriately named branch on your local machine. If you
 are part of the Recidiviz Team, you may push branches directly to the
 `recidiviz/recidiviz.com` remote. Otherwise, you may push branches to a [fork](https://help.github.com/articles/fork-a-repo/)
 of the repository.
 
-Before issuing a pull request, run our linting and test suite over your changes
-to ensure no regression, as described in [our Readme](README.md#linting--running-tests).
+Before issuing a pull request, test your changes locally at various screen size breaks.
 
-These same commands will be run by our continuous integration (CI) suite and
-your pull request will be "built against" automatically to show whether it
-passes. Stay ahead of the curve by running these throughout the development
-process to help you avoid any surprises.
-
-Related, include unit and//or integration tests to cover any new code. Update or
-remove existing test cases where appropriate.
-
-When you are passing linting and testing locally and are satisfied with the
+When you are satisfied with the
 current state of your work, submit a pull request against `master`. However,
 you can also submit Work In Progress ("WIP") pull requests to gather early
 feedback if you believe it would be helpful. If you do so, please format the
-title as "WIP: My title here."
+title as "WIP: <My title here>."
```

Comment:
These angular brackets are actually causing Github's Markdown renderer to lose this text. You might need to escape them.